Projects/UOContent/Commands/Possess.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using Server.Commands.Generic;
+using Server.Mobiles;
+
+namespace Server.Commands
+{
+    internal static class Swap
+    {
+        private static readonly List<Layer> SPECIAL_LAYERS = new() { Layer.Backpack, Layer.Bank, Layer.Mount };
+
+        private static List<Item> GetItemsFromLayers(IEnumerable<Item> items) =>
+            new List<Item>(items).FindAll(item => !SPECIAL_LAYERS.Contains(item.Layer));
```

Comment:
Linq is not efficient, so let's avoid using it:
```cs
private static List<Item> GetItemsOnMobile(Mobile m)
{
    var list = new List<Item>(_specialLayers.Count);
    for (var i = 0; i < _specialLayers.Count; i++)
    {
        var item = m.FindItemOnLayer(_specialLayers[i]);
        if (item != null)
        {
            list.Add(item);
        }
    }

    return list;
}
```

Reply:
I would probably not even make this function, avoid the list, and do everything inline

Reply:
Even though this says it is outdated, it is still relevant.